doc/Overview/Mvux/Advanced/Messaging.md

```diff
@@ -0,0 +1,162 @@
+---
+uid: Overview.Mvux.Advanced.Messaging
+---
+
+# Messaging
+
```

Comment:
Can you outline what the problem is that needs to be solved? ie data is being created/changed/deleted elsewhere in the application and need a way to update the current feed

Reply:
I'm still not seeing the "why"?  
Start by explaining the scenario that can't be handled by MVUX alone. This can be a simple example of what needs to be achieved and then discuss why messaging is an option to solve it